### PR TITLE
ref(sort): Add better priority to dropdown

### PR DESCRIPTION
**Pull request by ceorourke**
Add a new "better priority" option to the dropdown and populate some default params in the URL to make internal testing easier. Once we push to EA the new option will usurp the existing priority option.

### CODE REVIEW

**Comment by ceorourke** [comment]
@malwilley how can I get the URL to populate the [default query params](https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_group_index.py#L163-L176) that the endpoint sets? I'm adding this dropdown for internal testing and it'd be nice if the testers only had to edit some numbers, rather than needing to paste in a string of query params.

**Review by scttcper — APPROVED** [review]
i'd say to use `organization.features.includes('issue-list-better-priority-sort')` instead but it doesn't really matter

**Comment by malwilley** [comment]
@ceorourke I'd add some logic to this function: https://github.com/getsentry/sentry/blob/5b7d9d4345cb57e26a431d57b2d8b0611fb17ba3/static/app/views/issueList/overview.tsx#L851

You can do something like:

```ts
if (sort === 'betterPriority') {
  this.transitionTo({ logLevel: 10, ... })
}
```

File: static/app/views/issueList/overview.tsx [hunk 1]
```diff
@@ -853,8 +861,18 @@ class IssueListOverview extends Component<Props, State> {
       organization: this.props.organization,
       sort,
     });
-
-    this.transitionTo({sort});
+    if (sort === 'betterPriority') {
```
Comment (malwilley) [inline static/app/views/issueList/overview.tsx:864]: nit: could use the enum here instead (IssueSortOptions.BETTER_PRIORITY)

File: static/app/views/issueList/actions/sortOptions.tsx [hunk 2]
```diff
@@ -39,20 +54,20 @@ function IssueListSortOptions({onSelect, sort, query}: Props) {
...
-        label: getSortLabel(key),
-        details: getSortTooltip(key),
-      }))}
-      value={sortKey}
-      triggerProps={{
-        size: 'xs',
-        icon: <IconSort size="xs" />,
-      }}
-    />
+    <Feature features={['issue-list-better-priority-sort']}>
```
Comment (malwilley) [inline static/app/views/issueList/actions/sortOptions.tsx:57]: nit: wouldn't need the `Feature` or `getSortOptions` function if you did this at the top of the component:

```ts
const organization = useOrganization();
const hasBetterPrioritySort = organization.features.inculdes('issue-list-better-priority-sort')
const sortKeys = [
  ...(hasBetterPrioritySort ? [IssueSortOptions.BETTER_PRIORITY] : []),
  ...other stuff
]
```

**Review by malwilley — APPROVED** [review]

**Pull request merged**